contentcuration/contentcuration/static/js/edit_channel/import/views/ImportChannelList.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+
+  <div>
+    <div v-if="channelsAreLoading">
+      Channels are loading...
+    </div>
+    <div v-else-if="channels.length === 0">
+      No Channels available to import from
+    </div>
+
+    <ul v-else class="Channels">
+      <ImportListItem
+        v-for="channel in channels"
+        :key="channel.id"
+        :node="channel"
+        :isRoot="true"
+        :isFolder="true"
+        :isChannel="true"
+        :parentIsChecked="false"
+      />
+    </ul>
+  </div>
+
+</template>
+
+
+<script>
+
+const { mapState } = require('vuex');
+
+module.exports = {
+  components: {
+    ImportListItem: require('./ImportListItem.vue'),
+  },
+  computed: {
+    ...mapState('import', [
```

Comment:
Interesting. Even though the `...` spread operator is unnecessary, I like the convention of using it regardless so that it's easy to add in local computed props if desired.